css/index.css

```diff
@@ -71,10 +71,16 @@ select {
   -moz-appearance: none;
   -webkit-appearance: none;
 }
+
 option {
   color: #333;
 }
 
+body.dark select {
+  background: url("../images/dropdown-white.svg") no-repeat;
+  background-position: calc(100% - 14px) center;
+}
+
```

Comment:
We can probably apply any of these solutions mentioned here -> https://codepen.io/noahblon/post/coloring-svgs-in-css-background-images

but this would work for now.